app/controllers/navigation/Navigator.scala

```diff
@@ -258,7 +258,8 @@ object Navigator {
     case NatureOfTransaction                  => controllers.declaration.routes.TotalPackageQuantityController.displayPage
     case ProcedureCodes                       => controllers.declaration.routes.ItemsSummaryController.displayPage
     case DepartureTransport                   => controllers.declaration.routes.TransportLeavingTheBorderController.displayPage
-    case ExportItem                           => controllers.declaration.routes.PreviousDocumentsController.displayPage
+    case ExportItem                           => controllers.declaration.routes.PreviousDocumentsSummaryController.displayPage
+    case DocumentChangeOrRemove               => controllers.declaration.routes.PreviousDocumentsSummaryController.displayPage
```

Comment:
I didn't bother using the Navigator for back links from "Change" or "Remove" pages.  They always go back to the "Summary" page.    But this is fair enough I suppose.

Reply:
That's true, I can change it, I thought that we're using navigator for every back link